### ISSUE TITLE
Typescript is not supported

### ISSUE THREAD

**Issue by qkreltms** [llorentegerman/simple-flexbox#2]
**Note: I pull requested this issue see: https://github.com/llorentegerman/simple-flexbox/pull/3**

When I type npm install @types/simple-flexbox the "Not found" error occurred so we need to support type declaration for it.

Apart from it I made my own declaration file :
```ts
/// <reference types="react" />

declare module 'simple-flexbox' {
    declare class Column extends React.Component<ColumnProps> {}
    declare class Row extends React.Component<RowProps> {}

    declare interface ColumnProps {
        onClick?: (e: React.MouseEvent) => void;
        className?: string
        style?: React.CSSProperties
        flexGrow?: number
        wrap?: boolean
        flex?: string,
        flexShrink?: number
        flexBasis?: string
        breakpoints?: object
        reverse?: boolean
        children: any
        vertical?: 'start' | 'center' | 'end' | 'spaced' | 'space-between' | 'around' | 'space-around' | 'space-evenly'
        horizontal?: 'start' | 'center' | 'end' | 'stretch'
        justifyContent?: 'start' | 'flex-start' | 'center' | 'end' | 'flex-end' | 'spaced' | 'space-between' | 'around' | 'space-around' | 'space-evenly'
        alignItems?: 'start' | 'center' | 'end' | 'stretch' | 'baseline'
        alignSelf?: 'start' | 'center' | 'end' | 'stretch' | 'baseline'
        alignContent?: 'start' | 'flex-start' | 'center' | 'end' | 'flex-end' | 'spaced' | 'space-between' | 'around' | 'space-around' | 'stretch'
    }

    declare interface RowProps {
        onClick?: (e: React.MouseEvent) => void;
        className?: string
        style?: React.CSSProperties
        wrap?: boolean
        reverse?: boolean
        flex?: string
        flexGrow?: number
        flexShrink?: number
        flexBasis?: string
        breakpoints?: object
        children: any
        vertical?: 'start' | 'center' | 'end' | 'spaced' | 'baseline'
        horizontal?: 'start' | 'center' | 'end' | 'spaced' | 'space-between' | 'around' | 'space-around' | 'space-evenly'
        justifyContent?: 'start' | 'flex-start' | 'center' | 'end' | 'flex-end' | 'spaced' | 'space-between' | 'around' | 'space-around' | 'space-evenly'
        alignItems?: 'start' | 'center' | 'end' | 'stretch' | 'baseline' 
        alignSelf?: 'start' | 'center' | 'end' | 'stretch' | 'baseline' 
        alignContent?: 'start' | 'flex-start' | 'center' | 'end' | 'flex-end' | 'spaced' | 'space-between' | 'around' | 'space-around' | 'stretch' 
    }
}
```

**Steps to solve this issue**
1. I made index.d.ts in my root folder 
example:
```
typescript-playground
 |
 | ├── src
 | ├── public
 | ├── node_modules
 | ├── @types
 |         ├── simple-flexbox
 |                ├── index.d.ts
```

2. Add ```    "typeRoots": ["./node_modules/@types", "./@types"]``` in your tsconfig.json
example: 
```
{
  "compilerOptions": {
    "target": "es5",ue,
    "skipLibCheck": true,
    "..."
    "typeRoots": ["./node_modules/@types", "./@types"]
  },
}
```
3. Copy and paste the code that I wrote above into index.d.ts .

**Comment by llorentegerman** [comment]
@qkreltms thank you!

**Comment by llorentegerman** [comment]
Typescript support included in version **2.1.0**, see #3
